skrf/network.py

```diff
@@ -379,6 +385,11 @@ def __init__(self, file=None, name=None, comments=None, f_unit=None, **kwargs):
         self.noise = None
         self.noise_freq = None
 
+        if s_def not in S_DEFINITIONS:
+            raise ValueError('s_def parameter should be either:', s_def)
```

Comment:
this seems to not make sense.  shouldnt   you print `S_DEFINITIONS?`

Reply:
Yes indeed.